README.md

```diff
@@ -2,7 +2,7 @@
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy) [![Django CI](https://github.com/WPI-LNL/lnldb/actions/workflows/django-ci.yml/badge.svg)](https://github.com/WPI-LNL/lnldb/actions/workflows/django-ci.yml) [![Coverage Status](https://coveralls.io/repos/WPI-LNL/lnldb/badge.svg?branch=master&service=github)](https://coveralls.io/github/WPI-LNL/lnldb?branch=master) [![Documentation](https://readthedocs.org/projects/lnldb/badge/?version=latest&style=flat)](https://lnldb.readthedocs.io/en/latest/)
 
 ## Intro
-LNLDB now runs under Python3.x and Django 2.2 or later.
+LNLDB now runs under Python3.x (<= 3.10) and Django 2.2 or later.
```

Comment:
If we're doing this, should probably specify maximum supported django version (<3.2)